packages/token-bridge-sdk/src/util/index.ts

```diff
@@ -60,6 +60,16 @@ export async function getL1TokenData({
   l2Provider: Provider
   throwOnInvalidERC20?: boolean
 }): Promise<L1TokenData> {
+  // caching for tokens results
+  const l1TokenDataCache = JSON.parse(
+    sessionStorage.getItem('l1TokenDataCache') || '{}'
+  )
+  if (l1TokenDataCache?.[erc20L1Address]) {
+    // if found in cache - then return this only
+    console.log('**** CACHE HIT FOR ****', erc20L1Address)
+    return l1TokenDataCache?.[erc20L1Address]
```

Comment:
You can extract this to a variable, so once you check in the if, you don't need a `?` here anymore

Reply:
Done